tests/end_to_end/function-pointer/irix-o2-out.c

```diff
@@ -1,3 +1,3 @@
-s32 test(void *arg0, ? arg1) {
+s32 test(s32 (*arg0)(?, ? *), ? arg1) {
```

Comment:
This is kind of a cool example (even though mips2c incorrectly guessed that the function takes two args). The type here is recursive, the second arg has the type of the function pointer itself!